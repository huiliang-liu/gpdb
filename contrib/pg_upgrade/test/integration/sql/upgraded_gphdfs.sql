SELECT rolname, rolsuper, rolinherit, rolcreaterole, rolcreatedb, rolcatupdate, rolcanlogin, rolreplication, rolconnlimit, rolpassword, rolvaliduntil, rolconfig, rolcreaterextgpfd, rolcreaterexthttp, rolcreatewextgpfd FROM pg_roles WHERE rolname='gphdfsuser';
