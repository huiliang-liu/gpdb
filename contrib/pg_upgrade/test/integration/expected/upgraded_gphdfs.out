SELECT rolname, rolsuper, rolinherit, rolcreaterole, rolcreatedb, rolcatupdate, rolcanlogin, rolreplication, rolconnlimit, rolpassword, rolvaliduntil, rolconfig, rolcreaterextgpfd, rolcreaterexthttp, rolcreatewextgpfd FROM pg_roles WHERE rolname='gphdfsuser';
 rolname    | rolsuper | rolinherit | rolcreaterole | rolcreatedb | rolcatupdate | rolcanlogin | rolreplication | rolconnlimit | rolpassword | rolvaliduntil | rolconfig | rolcreaterextgpfd | rolcreaterexthttp | rolcreatewextgpfd 
------------+----------+------------+---------------+-------------+--------------+-------------+----------------+--------------+-------------+---------------+-----------+-------------------+-------------------+-------------------
 gphdfsuser | f        | t          | f             | f           | f            | f           | f              | -1           | ********    |               |           | f                 | f                 | f                 
(1 row)
